bin/qc_sentieon.pl

```diff
@@ -223,25 +223,21 @@ sub coverage_calc {
     my @cov;
     open( COV, $coverage_file );
     while( <COV> ) {
-
-        unless( /^Locus/ ) {
-            my $vals = <COV>;
-            my @a = split /\t/, $vals;
-            push @cov,$a[1];
-        }
-        
+        next if /^Locus/;
+        my @a = split /\t/;
+        push @cov, $a[1];
     }
-    my @sorted_cov = sort @cov;
+    close COV;
+
+    my @sorted_cov = sort { $a <=> $b } @cov;
```

Comment:
Before it did lexicographic sort, i.e. "100" came before "2"

This might screw up values quite a bit or?